taskcluster/docker/screenshots/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file is a copy of https://github.com/mozilla-mobile/fenix/blob/24596fd3cd493f38e1212e6ab2844e723565af9b/taskcluster/docker/base/Dockerfile. I changed the python part to be python 3 and to install the required dependencies. Nothing else has changed.